.expeditor/config.yml

```diff
@@ -0,0 +1,38 @@
+# Documentation available at https://expeditor.chef.io/docs/getting-started/
+---
+
+pipelines:
+ - verify:
+    description: Pull Request validation tests
+    public: true
+
+slack:
+ notify_channel: inspec-notify
+
+github:
+ delete_branch_on_merge: true
+ minor_bump_labels:
+  - "Version: Bump Minor"
+ version_tag_format: v{{version}}
+ release_branch:
+  - master
+ 
+changelog:
+ categories:
+  - "Type: New Resource": "New Resources"
+  - "Type: New Feature": "New Features"
+  - "Type: Enhancement": "Enhancements"
+  - "Type: Bug": "Bug Fixes"
+
+merge_actions:
+ - built_in:bump_version:
```

Comment:
This all looks good! I was just curious about the version bump. Does expeditor's `bump_version` require the `VERSION` file and then we use that to populate the Inspec yaml?

Reply:
The `bump_version` requires a version file, which is customizeable. https://expeditor.chef.io/docs/reference/built_in/#bump-version for more info

Reply:
Right and bump_version fires off .expeditor/update_version.sh through the only_if.